### PR TITLE
fix: Analytics tracking

### LINKED ISSUE #89: Fix staging/prod tracking
## Description

We have seen an increase in tracking on the staging environment post domain migration. This might be related to [this code section](https://github.com/safe-global/safe-landing-page/blob/dev/src/utils/googleAnalytics.ts#L11-L16) which marks everything that is not on `gnosis-safe` as staging (here called `GATSBY_GA_RINKEBY`).

## Steps to reproduce
 1. Open the landing page on staging
 2. Observe tracking on the `GATSBY_GA_MAINNET` project.

## Expected result
Tracking occurs on the `GATSBY_GA_RINKEBY` project.

## Obtained result

Tracking occurs on the `GATSBY_GA_MAINNET` project.

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves #89

## How it fixes it

- safe.global events are tracked to the main property on analytics
- Everything else is tracked to the staging property on analytics

## How to test

This can only be tested once it is deployed
1. Open the landing page on safe.global
2. Observe events being tracked in the main property on analytics

A more technical quick test to see that this works:
1. Open safe.global
2. Open the developer console
3. Paste `window.location.hostname`
4. This should always return "safe.global"
5. In this case events will be sent to the main property in analytics

### CODE REVIEW

File: src/utils/googleAnalytics.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const FOOTER_CATEGORY = 'FOOTER_CATEGORY'
 
 const getGoogleAnalyticsTrackingID = () => {
   return typeof window !== 'undefined' &&
-    window.location.hostname.split('.')[0] === 'gnosis-safe'
+    window.location.hostname.split('.')[0] === 'safe'
     ? process.env.GATSBY_GA_MAINNET
     : process.env.GATSBY_GA_RINKEBY
```
Comment (katspaugh) [inline src/utils/googleAnalytics.ts:15]: Please rename these vars to `GATSBY_GA_PROD` and `GATSBY_GA_STAGING`.

**Review by katspaugh — CHANGES_REQUESTED** [review]

File: src/utils/googleAnalytics.ts (hunk 1 above)
Comment (usame-algan) [reply src/utils/googleAnalytics.ts:15]: Are we also going to adjust the github secrets naming?

Comment (katspaugh) [reply src/utils/googleAnalytics.ts]: You can ask @luarx to do it.

Comment (usame-algan) [reply src/utils/googleAnalytics.ts]: I changed it in the code so we will have to adjust the secrets once this PR is merged.

File: src/utils/googleAnalytics.ts [hunk 2]
```diff
@@ -9,10 +9,11 @@ export const DEVELOPERS_CATEGORY = 'DEVELOPERS'
 export const FOOTER_CATEGORY = 'FOOTER_CATEGORY'
 
 const getGoogleAnalyticsTrackingID = () => {
-  return typeof window !== 'undefined' &&
+  return (typeof window !== 'undefined' &&
+    window.location.hostname.split('.')[0] === 'safe') ||
     window.location.hostname.split('.')[0] === 'gnosis-safe'
```
Comment (katspaugh) [inline src/utils/googleAnalytics.ts:14]: Why do you need to split the first part instead of just checking the full domain name?

Comment (usame-algan) [reply src/utils/googleAnalytics.ts:14]: Good question. I don't see any reason not to compare the hostname directly. Will change it accordingly

File: src/utils/googleAnalytics.ts [hunk 3]
```diff
@@ -9,10 +9,11 @@ export const DEVELOPERS_CATEGORY = 'DEVELOPERS'
 export const FOOTER_CATEGORY = 'FOOTER_CATEGORY'
 
 const getGoogleAnalyticsTrackingID = () => {
-  return typeof window !== 'undefined' &&
-    window.location.hostname.split('.')[0] === 'gnosis-safe'
-    ? process.env.GATSBY_GA_MAINNET
-    : process.env.GATSBY_GA_RINKEBY
+  return (typeof window !== 'undefined' &&
+    window.location.hostname === 'safe.global') ||
```
Comment (katspaugh) [inline src/utils/googleAnalytics.ts:13]: I think the parentheses are in the wrong place?

**Review by katspaugh — APPROVED** [review]

**Pull request merged**